app/createAccount/page.tsx

```diff
@@ -0,0 +1,44 @@
+"use client"
+import Link from "next/link";
+import { useState } from "react";
+import { Caveat } from 'next/font/google';
+import { createProfile } from "@/utils/scripts/profiles";
+import { useRouter } from "next/navigation";
+
+const caveat = Caveat({ subsets: ['latin'] })
+
+export default function Login() {
+
+  // Form data
+  const [name, setName] = useState("")
+  const [bio, setBio] = useState("")
+  const router = useRouter();
+
+  return (
+    <div className="2xl:bg-[url('/assets/backgrounds/loginBG_desktop.svg')] bg-[url('/assets/backgrounds/loginBG_laptop.svg')] bg-cover bg-center bg-no-repeat h-screen min-h-screen w-full relative flex justify-center items-center">
+        <div className="2xl:pt-6 pt-4 2xl:pl-10 pl-6 absolute top-0 left-0"><Link href="/" className={`${caveat.className} 2xl:text-6xl text-5xl text-primary font-medium`}>Minted</Link></div>
+        
+        {/* Log In Card */}
```

Comment:
idc about this but rename comment to create account card for consistency if you feel like it